org.uqbar.project.wollok/src/org/uqbar/project/wollok/WollokDsl.xtext

```diff
@@ -324,13 +330,25 @@ WCanExtendClass:
 // CLASSES
 
 WClass:
-	'class' name=ID ('inherits' parent=[WClass|QualifiedName])? '{'
+	'class' name=ID 
+		('inherits' parent=[WClass|QualifiedName])? 
+		('mixed with' mixins+=[WMixin|QualifiedName] (('and'|',') mixins+=[WMixin|QualifiedName])*)? 
+	'{'
 		(members+=WVariableDeclaration ';'?)*
 		(constructors+=WConstructor)*
 		(members+=WMethodDeclaration ';'?)*
 	'}'
 ;
 
+// MIXINS
+
+WMixin:
+	'mixin' name=ID '{'
```

Comment:
I do not love the word 'mixin', I think it means nothing for the student.
What about using a more significant name?... 'behavior' could be a name.